bigbluebutton-html5/imports/ui/components/nav-bar/component.jsx

```diff
@@ -88,6 +93,7 @@ class NavBar extends Component {
         <div className={styles.right}>
           <SettingsDropdown />
         </div>
+        {hasUnreadMessages ? this.soundForUnreadeMessages() : null}
```

Comment:
I dont think having this on the `render` method of the navBar makes sense. Proably should be moved to the App component